media/css/plugincheck/plugincheck.less

```diff
@@ -246,6 +245,48 @@ html[dir="rtl"] {
 .footer-newsletter-form {
     padding: 40px 20px;
 }
+.warning {
+    .span(8);
+    .offset(2);
+    clear: both;
+    display: inline-block;
+    margin-bottom: @baseLine;
+    padding: 0 35px;
+    font-style: italic;
+    text-align: center;
+    .font-size(18px);
+}
+// assume Fx users by default and hide nonFx user info.
+// assume js is enabled
+.not-supported,
+.enable-js {
+    display: none;
+}
+// Styles to show information for nonFx users
+.non-fx {
+    main,
+    .pluginfaq-container {
+        display: none;
+    }
+    .not-supported {
+        display: block;
+        #main-feature {
+            padding-bottom: 28px;
+        }
+        section {
+            background-color: #fff;
+            margin-bottom: 32px;
```

Comment:
Setting this `margin-bottom` and `padding` on the line below to `@baseLine` will allow the download button to fit it's container a bit neater at smaller viewport sizes.